### ISSUE TITLE
Option to disable the roundedness added in 2.16.41

### ISSUE THREAD

**Issue by ja231c** [WinMerge/winmerge#2491]
I find Firefox-style rounded tabs to be difficult to use, to the point that in Firefox I use custom css to restore the old, non-round ones.

Could an option be added to use the old non-round interface in WinMerge?

**Comment by sdottaka** [comment]
I haven't decided yet whether I'll add that option in the future, but is the display like the attached image what you want?

![image](https://github.com/user-attachments/assets/5335ab78-789b-451d-911e-507e39d01417)

**Comment by ja231c** [comment]
No, I want an option to disable all the interface changes done in that version, as in, I want it to look exactly like 2.16.40 does (including the bottom bar, which also lost it's seperators)

Only disabling the roundedness doesn't make it any better, it is difficult for me to tell where the tabs and other UI elements begin and end when there aren't any seperators.

Screenshots from 2.16.40 (which is what I want, wouldn't mind if it had slightly darker seperating lines though).
Top:
<img width="388" alt="winmerge_2-16-40" src="https://github.com/user-attachments/assets/c17e2291-80c4-44a6-aad8-9bb0216d2d44">
Bottom:
<img width="412" alt="winmerge_2-16-40_bottom" src="https://github.com/user-attachments/assets/aac02d53-7666-4890-bccd-398c2c15f3b8">

The tab seperating lines in the 2.16.40 were already a bit indistinct due to being slightly brighter than in the ones in windows dialogs (227 instead of 217) but it was much better than the current state of no lines at all.

And the centered file path is harder to read, with the whole path moving around when switching tabs when having multiple tabs with files from the same folder, with left aligned only the end of it changes and the position of the rest of the path stays the same.

That and one more line fits on screen in 2.16.40 than in 2.16.41+.

**Comment by lededev** [comment]
I think it's a better choice for tabs to automatically have multiple background colors than to change them back to the original version, and the background area is larger, which is easier to distinguish than separating the tabs with vertical lines.
To avoid too many colors, we can use two colors alternately, similar to the alternating background colors of some lists. The currently active tab should be significantly different from the two colors above.
![image](https://github.com/user-attachments/assets/2feb331a-5830-4f56-b3f8-fcf0781c024c)

**Comment by ganego** [comment]
I agree. I went back to an older version.  
Another gamebreaker for me is that when WinMerge is maximized I cannot just put the mouse cursor in the upper right corner of the screen to close the program. It won't trigger the 'x'. Yes this alone makes me not use the new version.

**Comment by sdottaka** [comment]
> Another gamebreaker for me is that when WinMerge is maximized I cannot just put the mouse cursor in the upper right corner of the screen to close the program. It won't trigger the 'x'.

Oh, you can close it by moving the mouse pointer to the center of the close button, but if you move it to the corner, the mouse pointer icon changes to a window resize one. I'll fix that later.
If you uncheck the View → Tab Bar → On Title Bar menu item, you can click the corner of the close button.

![image](https://github.com/user-attachments/assets/17fb165a-67d1-4308-a6e0-30ef5ee01964)
